packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/Abs_L2DepositedToken.sol

```diff
@@ -139,66 +140,98 @@ abstract contract Abs_L2DepositedToken is iOVM_L2DepositedToken, OVM_CrossDomain
 
     /**
      * @dev initiate a withdraw of some tokens to the caller's account on L1
-     * @param _amount Amount of the token to withdraw
+     * @param _amount Amount of the token to withdraw.
+     * @param _data Data to forward to L1.
+     * @param _l1Gas Gas limit for the provided message.
      */
     function withdraw(
-        uint _amount
+        uint _amount,
+        bytes calldata _data,
+        uint32 _l1Gas
     )
         external
         override
         virtual
         onlyInitialized()
     {
-        _initiateWithdrawal(msg.sender, _amount);
+        _initiateWithdrawal(
+            msg.sender,
+            msg.sender,
+            _amount,
+            _data,
+            _l1Gas
```

Comment:
I can't see the l1Gas being used anywhere along the call logic.

Reply:
https://github.com/ethereum-optimism/optimism/pull/824/files#diff-3839942a735e09a99e231eb4c4d40155b62d9c2e73fe87ce9a4ed541fac2399cR226 :)

Reply:
It has no logic associated with it though